pom.xml

```diff
@@ -715,7 +715,7 @@
       <dependency>
         <groupId>io.sf.carte</groupId>
         <artifactId>css4j</artifactId>
-        <version>3.9.1</version>
+        <version>4.0</version>
```

Comment:
I guess this is not part of the PR?

Reply:
When I tried to build (to take screenshots and update), I had an issue, and did a fetch - rebase - push force with lease.
However the two last commits in the history of the PR as of now are still here...